app/views/spree/admin/shared/_routes.html.erb

```diff
@@ -1,10 +1,13 @@
 <script>
   Spree.routes = <%== {
-    :variants_search       => admin_search_variants_path(:format => 'json'),
+    :variants_search       => spree.api_variants_path(:format => 'json'),
```

Comment:
I think it wont work, it's because we need to scope variants when we search. Scoping means to transform variants into variant_overrides (variants of one supplier sold by the current hub). What's the problem with our route?

Reply:
this can be addressed as part of #2416